webpack.config.js

```diff
@@ -108,11 +108,6 @@ module.exports = {
             "matrix-js-sdk": path.resolve('./node_modules/matrix-js-sdk'),
         },
     },
-    externals: {
```

Comment:
I am not sure I follow why this was removed... It seems unrelated to the rest of the changes. Is it something outdated that's no longer needed?

Reply:
The webpack side doesn't import electron anymore from what I can see. Means that it doesn't need to be declared anymore in the webpack config. (webpack side == everything that actually is riot)